### PR TITLE
cardano-testnet: QoL changes

### PR DESCRIPTION
**Pull request by smelc**
[edit] QoL improvements, because most of this PR's content has been concurrently implemented in https://github.com/IntersectMBO/cardano-node/pull/6007

### CODE REVIEW

File: cardano-testnet/src/Parsers/Cardano.hs [hunk 1]
```diff
@@ -65,7 +65,7 @@ pCardanoTestnetCliOptions envCli = CardanoTestnetOptions
 pNumSpoNodes :: Parser [TestnetNodeOptions]
 pNumSpoNodes =
   OA.option
-     ((`L.replicate` SpoTestnetNodeOptions Nothing []) <$> auto)
+     ((`L.replicate` TestnetNodeOptions Nothing []) <$> auto)
```
Comment (Jimbo4350) [inline cardano-testnet/src/Parsers/Cardano.hs:70]: We should keep the differentiation between SPOs and relays.

**Review by Jimbo4350 — COMMENTED** [review]
One minor comment.

**Comment by smelc** [comment]
The point of this PR was actually implemented in https://github.com/IntersectMBO/cardano-node/pull/6007, so this one becomes mostly empty.

**Review by carbolymer — APPROVED** [review]

File: cardano-testnet/src/Parsers/Cardano.hs [hunk 2]
```diff
@@ -65,6 +65,8 @@ pCardanoTestnetCliOptions envCli = CardanoTestnetOptions
 
 pNumSpoNodes :: Parser [TestnetNodeOptions]
 pNumSpoNodes =
+  -- We don't support passing custom node configurations files on the CLI.
```
Comment (Jimbo4350) [inline cardano-testnet/src/Parsers/Cardano.hs:68]: :+1:

File: cardano-testnet/src/Testnet/Start/Cardano.hs [hunk 3]
```diff
@@ -144,7 +144,7 @@ getDefaultShelleyGenesis asbe maxSupply opts = do
 -- > │   └── README.md
 -- > ├── drep-keys
 -- > │   ├── drep{1,2,3}
--- > │   │   └── drep.{skey,drep.vkey}
+-- > │   │   └── drep.{skey,vkey}
```
Comment (Jimbo4350) [inline cardano-testnet/src/Testnet/Start/Cardano.hs:147]: How did you notice to change this?

**Review by Jimbo4350 — APPROVED** [review]

File: cardano-testnet/src/Testnet/Start/Cardano.hs (hunk 3 above)
Comment (smelc) [reply cardano-testnet/src/Testnet/Start/Cardano.hs:147]: It's @carbolymer who did this change, I think after seeing my (late) review on https://github.com/IntersectMBO/cardano-node/pull/6007 :wink:

**Pull request merged**